app/views/registrations/edit.html.erb

```diff
@@ -14,15 +14,43 @@
              url: registration_path(resource),
              html: { class: 'pss-admin-form' }) do |f| %>
 
-  <%= f.label :email %><br />
-  <%= f.email_field :email, autofocus: true %>
-
-  <%= f.label :password %>
-  <%= f.password_field :password, autocomplete: "off" %>
-  <div><i>(leave blank if you don't want to change it)</i></div>
-
-  <%= f.label :password_confirmation %><br />
-  <%= f.password_field :password_confirmation, autocomplete: "off" %>
+  <fieldset>
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
+  </fieldset>
+
+  <% if current_admin.id == resource.id %>
+  <fieldset>
+    <%= f.label :password %>
+    <%= f.password_field :password, autocomplete: "off" %>
+    <div class="pss-admin-form-comment">
+      <i>(leave blank if you don't want to change it)</i>
+    </div>
+    <%= f.label :password_confirmation %>
+    <%= f.password_field :password_confirmation, autocomplete: "off" %>
+  </fieldset>
+  <% else %>
+  <p class="pss-admin-form-pwmessage">
+    To change the password, direct the user to go to the
+    password reset page at
+    <%= Settings.app_scheme + Settings.app_host + new_admin_password_path %>
+  </p>
+  <% end %>
+
+  <% if current_admin.manager? %>
+  <fieldset class="status-buttons">
+    <legend>Status</legend>
+    <%= f.radio_button :status, :reviewer,
+                       checked: resource.status == 'reviewer' %>
+    <%= f.label :status_reviewer, "reviewer" %>
+    <%= f.radio_button :status, :editor,
+                       checked: resource.status == 'editor' %>
+    <%= f.label :status_editor, "editor" %>
+    <%= f.radio_button :status, :manager,
+                       checked: resource.status == 'manager' %>
+    <%= f.label :status_manager, "manager" %>
+  </fieldset>
```

Comment:
Could we add some brief text explaining the difference permissions afforded to reviewer, editor, and manager?